playbooks/amc.yml

```diff
@@ -30,6 +30,7 @@
     - role: mongo
       when: "'localhost' in EDXAPP_MONGO_HOSTS"
     - { role: 'rabbitmq', rabbitmq_ip: '127.0.0.1' }
+    # Do we need need edxapp 2 times here????
```

Comment:
This would be good to note here. I can see this question popping up again in the future.